.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+
+
+
+## Code Review Checklist
+ * Does the PR describe what changes are being made?
+ * Does the code follow the [Effective Go style guide](https://golang.org/doc/effective_go.html)?
```

Comment:
link to our guide?

Reply:
which guide?

Reply:
yeah i was wondering the same, there is this storj style guide https://github.com/storj/docs/blob/master/code/Style.md and also for testing https://github.com/storj/docs/blob/master/code/Testing.md